rs/nervous_system/integration_tests/tests/advance_sns_target_version.rs

```diff
@@ -67,9 +68,10 @@ async fn test_get_upgrade_journal() {
     // Step 1: Check that the upgrade journal contains the initial version right after SNS creation.
     let mut expected_upgrade_journal_entries = vec![];
     {
-        expected_upgrade_journal_entries.push(Event::UpgradeStepsRefreshed(
-            UpgradeStepsRefreshed::new(vec![initial_sns_version.clone()]),
-        ));
+        expected_upgrade_journal_entries.push(Event::UpgradeStepsReset(UpgradeStepsReset::new(
+            "redacted".to_string(),
```

Comment:
???

Reply:
Can you change the text to something that makes this clear?

Reply:
Done